src/extensions/default/ESLint/domain.js

```diff
@@ -0,0 +1,132 @@
+/*eslint no-process-env:0*/
+/*global require, exports*/
+
+(function () {
+  'use strict';
+
+  var fs = require('fs');
+  var CLIEngine = require('eslint').CLIEngine;
+  var cli = new CLIEngine();
+  var currentProjectRoot = null;
+  var domainName = 'brackets-eslint';
+  var domainManager = null;
+  var noop = function () {};
+  var globalPackagesAvailable = false;
+
+  function _setProjectRoot(projectRoot) {
+    var opts = { useEslintrc: true };
+    var configPath;
+    var rulesDirPath;
+    var ignorePath;
+
+    if (projectRoot) {
```

Comment:
@zaggino hah, wow, I wrote the thing in as async with a promise, noticed that it probably should be sync instead and then I don't know what happened. Waaaay too little coffee for morning coding. :coffee: